discovery-handlers/onvif/src/discovery_impl.rs

```diff
@@ -183,7 +183,7 @@ pub mod util {
         scopes: Option<&FilterList>,
     ) -> Vec<String> {
         let response_envelope =
-            yaserde::de::from_str::<to_deserialize::Envelope>(&discovery_response);
```

Comment:
are the removal of all of these borrowing `&`s coming from clippy?  we were really overusing `&`

Reply:
Yeah, especially in our tests. I didn't even address them all. I ran the following to auto-fix some:
```
rustup override set nightly
cargo clippy --fix -Z unstable-options --allow-dirty
```
Note: it requires nightly to run, so unset the nightly version like so: `rustup override unset`